config.json

```diff
@@ -37,6 +37,10 @@
     "last 3 versions"
   ],
   "settings": {
+    "use_custom_background": false,
```

Comment:
Should these settings be scoped to the checkout somehow, maybe with a prefix? I take it nested objects aren't allowed?